src/main/resources/application.properties

```diff
@@ -179,3 +179,6 @@ ccd.conditional-apis.case-assigned-users-and-roles.enabled=${CCD_CONDITIONAL_API
 # (Note SecurityLoggingFilter logs at debug)
 security.logging.filter.path.regex=${DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX:.*}
 
+caseworker.role.regex=${CASEWORKER_ROLE_REGEX:caseworker.+}
```

Comment:
can you add these under the section: 
`# List of roles which are comma separated`
and prefix them with:
`ccd.access-control`
please